code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -713,7 +782,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 					if(last_open <= world.time - 20) //Spam limiter.
 						last_open = world.time
 						user.visible_message(SPAN_DANGER("After a few attempts, <b>[user]</b> manages to light \the [src], they however burn their finger in the process."), range = 3)
-			set_light(2, 1, l_color = LIGHT_COLOR_LAVA)
+			set_flame_light()
```

Comment:
unnecessary proc. just put the set_light here.

Reply:
Proc's gone, chief.